src/Nancy.Bootstrappers.StructureMap/StructureMapNancyBootstrapper.cs

```diff
@@ -1,3 +1,5 @@
+using Nancy.Configuration;

```

Comment:
Dude, I'm serious. This is the only project in the .NET space I've contributed to that has these conventions. If you want me to get things right the first time, the tooling has to help me, i.e. using a .DotSettings :smile: